wgsl/index.bs

```diff
@@ -347,6 +347,13 @@ A vector type is a <dfn dfn>numeric vector</dfn> type if its component type is a
                        These may only appear in specific contexts.<br>
 </table>
 
+An array element type must be one of:
```

Comment:
I assume this set of types is going to show up a lot. I.e. the same should apply to structure members, function arguments, etc. Can we have a common definition for this?

Reply:
We have two cases in this PR.

The closest analog in SPIR-V is "concrete" type.  (That also covers a physical pointer, which is not in WSGL).   I think we should keep an eye on this and if we get a lot more cases then we'll add a definition.